indico/web/client/js/react/components/category_navigator/style.scss

```diff
@@ -0,0 +1,262 @@
+// This file is part of Indico.
+// Copyright (C) 2002 - 2024 CERN
+//
+// Indico is free software; you can redistribute it and/or
+// modify it under the terms of the MIT License; see the
+// LICENSE file for more details.
+
+@use 'partials/icons';
+@use 'partials/spinner';
+@use 'design_system';
+
+%category-list-item {
+  @extend %flex-split;
+
+  min-height: 3.2em;
+  padding: var(--content-gap-within-inline) var(--content-gap-normal);
+
+  background: var(--background-base);
+  border-left: 3px solid var(--container-separator-color);
+  border-bottom: 1px solid var(--container-separator-color);
+
+  > * {
+    padding: 0.5em;
```

Comment:
Still feels a big like wasting space to me on paddings... Is there any a11y problem w/ going even lower here?

Reply:
It's generally not a good idea to remove vertical separation when there is ample horizontal separation between elements. But no, there's no critical a11y failure, it just makes for suboptimal readability.

Reply:
I think the alternative would be making the dialog bigger so you can see more at once. But I guess you agree that seeing only very few items at a time and having to scroll a lot is suboptimal?

Reply:
Increasing the dialog size will help, sure. Additionally, we can also implement a simplified view that has less elements, and that list can be squashed tighter. That's been the usual pattern in apps I've used.